core/src/main/java/org/opensearch/sql/calcite/udf/UserDefinedAggFunction.java

```diff
@@ -0,0 +1,15 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package org.opensearch.sql.calcite.udf;
+
+public interface UserDefinedAggFunction<S extends Accumulator> {
+  S init();
+
+  Object result(S accumulator);
```

Comment:
in UDF, the method name is `eval`, should we change to `eval` either?

Reply:
For the function name of agg function, it's defined in calcite here https://github.com/apache/calcite/blob/1793ba79a328c61fb42842f443334cc1353c985f/core/src/main/java/org/apache/calcite/schema/impl/AggregateFunctionImpl.java#L91. We cannot modify them. I will left comment.